evergreen/system_perf/master/genny_tasks.yml

```diff
@@ -770,6 +770,54 @@ buildvariants:
 - name: perf-3-node-replSet-limited-10000-query-stats.arm.aws.2023-11
   tasks:
   - name: non_search_hybrid_scoring
+- name: perf-1-node-replSet-default-sharding.arm.aws.2023-11
```

Comment:
So we're the first ones to want to use any of these variants for perf tests? Interesting

Relatedly, how did you pick this list? I can't tell from the names here exactly what they represent, but they're all basically 1-shard variants?

Reply:
We're the first ones to add Genny tasks to these variants, they're currently defined in dsi https://github.com/10gen/dsi/blob/3a032e9a89abcafc9a8400b3e2b0d4cee182b97e/evergreen/system_perf/master/master_variants.yml#L1493

Unfortunately the names here are not immediately obvious that they're related to invisible sharding (/all clusters sharded) but you can see the comment in the link above for why that's the case.